common/network-yarn/src/main/java/org/apache/spark/network/yarn/YarnShuffleService.java

```diff
@@ -243,7 +243,6 @@ public void initializeApplication(ApplicationInitializationContext context) {
     String appId = context.getApplicationId().toString();
     try {
       ByteBuffer shuffleSecret = context.getApplicationDataForService();
-      logger.info("Initializing application {}", appId);
```

Comment:
I don't think this is to verbose personally.  We do log when initializeContainer so I guess its ok to remove this.  But if you remove this I think we should remove the one in stopApplication as well since they go hand in hand.